public/js/index.js

```diff
@@ -1,3 +1,47 @@
+var url = "http://api.hostip.info/get_json.php?ip=24.66.128.2";
+let city;
+let exampleCityEl;
+
+
+function getScript(url, success) {
+      var script = document.createElement('script');
+      script.src = url;
+      var head = document.getElementsByTagName('head')[0],
+          done = false;
+      // Attach handlers for all browsers
+      script.onload = script.onreadystatechange = function() {
+        if (!done && (!this.readyState
+             || this.readyState == 'loaded'
+             || this.readyState == 'complete')) {
+          done = true;
+          success();
+          script.onload = script.onreadystatechange = null;
+          head.removeChild(script);
+        }
+      };
+      head.appendChild(script);
+  };
+
+
+getScript('https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.0/jquery.min.js',function() {
+        console.log("Yay jQuery is ready");
+    }); 
+
+$(document).ready(function() {
+    $.get(url, function(data) {
+        $('#city').text(data.city);
+        city = data.city;
+    });
+});
+
+function myFunction() {
```

Comment:
This really should be implemented as a callback. We can't rely on a specified amount of time to run this; it should executed as soon as the dependent information is returned